app/assets/stylesheets/_deprecated/_nocss.scss

```diff
@@ -1597,111 +1090,27 @@ $px4:   4px;
 .#{$pre}caps { text-transform: uppercase }
 
 .#{$pre}strong { font-weight:bold }
-.#{$pre}fw-500 { font-weight:500 }
-.#{$pre}fw-600 { font-weight:600 }
 
 .#{$pre}italic { font-style: italic }
 
 .#{$pre}fs0    { font-size:$r0 }
 .#{$pre}fs0_9  { font-size:$r0_9 }
-.#{$pre}fs1    { font-size:$r1 }
-.#{$pre}fs1_1  { font-size:$r1_1 }
-.#{$pre}fs1_2  { font-size:$r1_2 }
-.#{$pre}fs1_3  { font-size:$r1_3 }
-.#{$pre}fs1_4  { font-size:$r1_4 }
-.#{$pre}fs1_5  { font-size:$r1_5 }
-.#{$pre}fs1_6  { font-size:$r1_6 }
-.#{$pre}fs1_7  { font-size:$r1_7 }
-.#{$pre}fs1_8  { font-size:$r1_8 }
-.#{$pre}fs1_9  { font-size:$r1_9 }
-.#{$pre}fs2    { font-size:$r2 }
-.#{$pre}fs2_5  { font-size:$r2_5 }
+.#{$pre}h5    { font-size:$r1 }
+.#{$pre}h5_1  { font-size:$r1_1 }
+.#{$pre}h5_2  { font-size:$r1_2 }
+.#{$pre}h5_3  { font-size:$r1_3 }
+.#{$pre}h5_4  { font-size:$r1_4 }
+.#{$pre}h5_5  { font-size:$r1_5 }
+.#{$pre}h5_6  { font-size:$r1_6 }
```

Comment:
Name of variable `r1_6` should be written in all lowercase letters with hyphens instead of underscores<br>Colon after property should be followed by one space<br>Opening curly brace `{` should be preceded by one space<br>Declaration should be terminated by a semicolon